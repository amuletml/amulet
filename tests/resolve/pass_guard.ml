let f = function
| x when x > 2 -> ()
