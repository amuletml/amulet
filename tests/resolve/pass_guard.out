pass_guard.ml[4:12 ..4:12]: error
  Variable not in scope: `>`

  Arising from use of the expression
  │ 
4 │ | x when x > 2 -> ()
  │            ^
